### DISCUSSION TITLE
Single Stranded DNA

### DISCUSSION THREAD

**Discussion by nealamo** [BradyAJohnston/MolecularNodes#151] [Q&A] [answered]
Hi Brady ,
First of all congratulations for your great Blender add-on.
I am working with Blender version 3.4.1 and Molecular Nodes version 2.2.1 
I am trying to find the way to create single stranded DNA (I saw that in previous versions but I am not able to find the way. The nodes that I have available do not have this option.
Am I missing something? Is there a way to get that with this version?
Thanks
Eduardo

**Comment by BradyAJohnston** [comment, marked as answer]
The outputted double stranded DNA will have the named attribute `chain_id` which you can use to delete one strand and keep the other using the `Delete Geometry` node. I can potentially add this back in to the node itself, as it seems like lots of people liked the easy access to a single strand.

**Reply by nealamo** [reply]
Thank you very much Brady. Already done !

**Reply by drquero** [reply]
Hi! Could you send a screenshot showing how you solved the issue? I'm having the same problem and I don't manage to solve it with the shared solution (I'm very new using Blender ...). Thanks! :)

**Reply by BradyAJohnston** [reply]
@drquero This is how you can achieve it. Ensure the `Separate Geometry` is switched to `Instance` and not `Point`: 
![CleanShot 2023-12-05 at 11 25 54@2x](https://github.com/BradyAJohnston/MolecularNodes/assets/36021261/c54bad3b-4bf3-4d6d-b863-45b7fcb058b5)

**Reply by drquero** [reply]
Worked perfectly! Thanks again!
